collectd.yaml

```diff
@@ -0,0 +1,179 @@
+package:
+  name: collectd
+  version: 5.12.0
+  epoch: 0
+  description: "The system statistics collection daemon."
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cairo-dev
+      - curl-dev
+      - eudev-dev
+      - flex
+      - gdk-pixbuf-dev
+      - gpsd-dev
+      - harfbuzz-dev
+      - hiredis-dev
+      - i2c-tools-dev
+      - iptables-dev
+      - jansson-dev
+      - libatasmart-dev
+      - libdbi-dev
+      - libgcrypt-dev
+      - libmemcached-dev
+      - libmicrohttpd-dev
+      - libmnl-dev
+      - libmodbus-dev
+      - libnotify-dev
+      - liboping-dev
+      - libpcap-dev
+      - libpq-16
+      - librdkafka-dev
+      - libtool
+      - libxml2-dev
+      - lm-sensors-dev
+      - mariadb-connector-c-dev
+      - mariadb-dev
+      - mosquitto-dev
+      - net-snmp-dev
+      - openipmi-dev
+      - openjdk-17-default-jdk
+      - openldap-dev
+      - openssl-dev>3
+      - owfs-dev
+      - pango-dev
+      - patchelf
+      - perl-dev
+      - pkgconf
+      - pkgconf-dev
+      - postgresql-16-dev
+      - python3-dev
+      - rabbitmq-c
+      - rabbitmq-c-dev
+      - riemann-c-client-dev
+      - rrdtool-dev
+      - varnish
+      - yajl-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/collectd/collectd.git
+      tag: collectd-${{package.version}}
+      expected-commit: 3f935d017e81b7eaf84c9df421490230845dc5c0
+
+  - runs: |
+      ./build.sh
+
+      if "${{build.arch}}" == "x86_64"; then
+        _flags="--disable-turbostat"
+      elif "${{build.arch}}" == "aarch64"; then
+        _flags="--disable-java"
+      fi
```

Comment:
```suggestion
```

Does this build without all these? I see a `--disable-turbostat` flag further down. I'd try without and drop the `$_flags` bit at the end of configure as a test

Reply:
I'd like to remove these too if we can. I don't like the idea of having different features enabled per-arch, since we basically only test things on x86_64. If someone used the aarch64 image expecting Java to work, they would be surprised.

Reply:
removed, yes.